src/apps/tenderize/common/tenderize.farm.contract-position-fetcher.ts

```diff
@@ -0,0 +1,51 @@
+import { Inject } from '@nestjs/common';
+
+import { APP_TOOLKIT, IAppToolkit } from '~app-toolkit/app-toolkit.interface';
+import { GetDataPropsParams, GetTokenBalancesParams } from '~position/template/contract-position.template.types';
+import {
+  SingleStakingFarmDataProps,
+  SingleStakingFarmTemplateContractPositionFetcher,
+} from '~position/template/single-staking.template.contract-position-fetcher';
+
+import { TenderizeContractFactory, TenderFarm } from '../contracts';
+
+import { TenderizeTokenDefinitionsResolver } from './tenderize.token-definition-resolver';
+
+export abstract class FarmTokenFetcher extends SingleStakingFarmTemplateContractPositionFetcher<
```

Comment:
Since it's a contract position, I would suggest you update the name to something `TenderizeFarmContractPositionFetcher`